test/integration/helpers/strings.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var _ = require('lodash');
+
+exports.startCase = function(str) {
+  return str && _.startCase(str) || '';
```

Comment:
Can we just call `_.startCase` directly? It handles null / undefined values already (returning `''`)